unit-tests/func/MUR/l500/test-Reading-MUR-option-off.cpp

```diff
@@ -0,0 +1,48 @@
+// License: Apache 2.0. See LICENSE file in root directory.
```

Comment:
filename is using uppercase...

Reply:
and your directory is in uppercase, too

Reply:
I suggest `test-get-disabled.cpp`?

Reply:
> filename is using uppercase...

Apparently Git is case insensitive, I fixed it.

Reply:
Merged with "test-bad-conditions" and added test for not streaming